src/library/index.ts

```diff
@@ -0,0 +1,134 @@
+import { experimental, strings } from '@angular-devkit/core';
+import {
+    Rule,
+    SchematicsException,
+    Tree,
+    apply,
+    applyTemplates,
+    chain,
+    mergeWith,
+    move,
+    url,
+    filter,
+    noop
+} from '@angular-devkit/schematics';
+
+import { parseName } from '../utilities/parse-name';
+import { buildDefaultPath, getProject } from '../utilities/project';
+import { validateName } from '../utilities/validation';
+
+interface LibraryOptions {
+    name: string;
+    path: string;
+    project: string;
+    prefix: string;
+    skipTests?: boolean;
+}
+
+interface NxWorkspaceSchema {
+    npmScope: string;
+    affected: {
+      [key:string]: any;
+    },
+    implicitDependencies: {
+        [key:string]: any;
+    },
+    tasksRunnerOptions: {
+        [key:string]: any;
+    },
+    projects: {
+        [key:string]: any;
+    }
+}
+
+export default function (options: LibraryOptions): Rule {
+    const angularConfigFile = '/angular.json';
```

Comment:
Isn't this supposed to be the `project-configs-builder.ts`? (even though it's in our roadmap to remove it)
Otherwise the `angular.json` will be overwritten

Reply:
Actually, removing `project-configs-builder.ts` should be part of the scope of this PR. I'll add that change and all other requirements missing.
See [here](https://avrios.atlassian.net/browse/AV-24340)

Reply:
Ah, I thought it was a separate ticket 😁 
Nice 💪